tests/integration/test_as_file_against_aleth.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import requests
+
+# The following tests invoke dopple as a pure file (as opposed to a package installed through pip)
+# and test it's functionality against aleth.
+
+
+@pytest.mark.trio
+async def test_run_and_get_info(dopple_as_file_on_aleth, aleth_ipc_path):
+    ret = requests.get("http://127.0.0.1:8545")
+    assert aleth_ipc_path in ret.text
+    assert "connected: True" in ret.text
+
+
+@pytest.mark.trio
+async def test_can_request_data(dopple_as_file_on_aleth):
+    data = {
+        "jsonrpc": "2.0",
+        "method": "eth_getBlockByNumber",
+        "params": ["0x0", True],
+        "id": 1,
+    }
+    ret = requests.post("http://127.0.0.1:8545", json=data)
+    assert (
+        "0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3" in ret.text
```

Comment:
Maybe worth making a quick note as to what this value actually represents?  I assume it is the block hash of the geneses block but maybe worth commenting so that nobody has to look it up.

Or you could change the name of the test to `request_genesis_block_by_number`